e2e/playwright.config.ts

```diff
@@ -6,9 +6,9 @@ const config: PlaywrightTestConfig = {
     forbidOnly: Boolean(process.env.CI),
     retries: process.env.CI ? 2 : 1,
     workers: 4,
-    timeout: 60 * 1000,
+    timeout: 30 * 1000,
     expect: {
-        timeout: 30 * 1000,
+        timeout: 10 * 1000,
```

Comment:
Unrelated but lowering some timeouts after the latest issue which took way too long cause of these being so high.